app/assets/javascripts/organisation-autocomplete.js

```diff
@@ -5,16 +5,118 @@ window.GOVUKAdmin.Modules = window.GOVUKAdmin.Modules || {};
+              .find('option[value="' + $(option).attr('value') + '"]')
+              .attr('selected', 'selected')
+          }
+        });
+
+      enableAutocomplete();
+
+      function createAutocompleteTemplate() {
+        var $select = organisationSelects().first();
+        $select.removeAttr('multiple');
+        var $wrapper = wrapper($select);
+        var $template = $wrapper.clone();
+        $template.find('[selected]').removeAttr('selected');
+        return $template.prop('outerHTML');
+      }
+
+      function addOrganisationButton() {
+        return $(
+          '<button type="button" ' +
+          '        id="add-organisation" ' +
+          '        class="add-organisation js-add-organisation">' +
+          '  Add another organisation' +
+          '</button>'
+        ).click(addOrganisation);
+      }
+
+      function removeOrganisationButton() {
+        return $(
+          '<button type="button" ' +
+          '        class="remove-organisation js-remove-organisation">' +
+          '  <span class="glyphicon glyphicon-remove"' +
+          '        aria-hidden="true"></span>' +
+          '  <span class="sr-only">Remove organisation</span>' +
+          '</button>'
+        ).click(removeOrganisation);
+      }
+
+      function enableAutocomplete() {
+        organisationSelects().each(function (index, select) {
+
+          // When no option is selected, the selectedIndex property will return
+          // -1. We have a blank option at index 0. Selecting the blank option
+          // circumvents a regression in the accessible autocomplete component
+          // where it assumes `selectedIndex` is always a positive number.
+          if (select.selectedIndex === -1) {
+            select.selectedIndex = 0;
+          }
+
+          enhanceSelectElement(select);
+        });
+      }
+
+      function organisationSelects() {
+        return $element.find('.js-organisation-select');
+      }
+
+      function enhanceSelectElement(select) {
+        accessibleAutocomplete.enhanceSelectElement({
+          selectElement: select,
+          showAllValues: true,
+          autoselect: false,
+          dropdownArrow: dropdownArrow
+        });
+
+        wrapper($(select)).append(removeOrganisationButton());
+      }
+
+      function addOrganisation() {
+        var $newSelectWrapper = $(autocompleteTemplateHtml);
+        enhanceSelectElement($newSelectWrapper.find('select')[0]);
+        $newSelectWrapper.insertAfter(lastSelectWrapper());
+      }
+
+      function removeOrganisation(event) {
+        var $removeButton = $(event.target);
+        var $organisationSelectWrapper = wrapper($removeButton);
+
+        if (organisationSelects().length === 1) {
+          clearAutocompleteAndSelect($organisationSelectWrapper);
+        } else {
+          $organisationSelectWrapper.remove();
+        }
+      }
+
+      function clearAutocompleteAndSelect($organisationSelectWrapper) {
```

Comment:
Sorry, I forgot to remove this method when we got rid of the "X" on the first select, so I believe this is unused now? However, we could repurpose it to fire on `change` for the `input` fields, to address the bug where backspacing the `input` doesn't select the blank option?

Reply:
I think we should fix that with https://trello.com/c/AvtSdooL/528-organisations-cant-be-de-selected